ui/admin/tests/e2e/helpers/boundary-ui.js

```diff
@@ -431,6 +431,9 @@ exports.createSshTargetWithAddressEnt = async (page) => {
   await page.getByRole('group', { name: 'Type' }).getByLabel('SSH').click();
   await page.getByLabel('Target Address').fill(process.env.E2E_TARGET_ADDRESS);
   await page.getByLabel('Default Port').fill(process.env.E2E_TARGET_PORT);
+  const workerTag = `"${process.env.E2E_WORKER_TAG_EGRESS}" in "/tags/type"`;
+  await page.getByLabel('Ingress worker filter').click();
```

Comment:
I think this should be `Egress`

- I'd like to make this its own method so that you don't always need a worker filter
- I'm also thinking we should refactor these methods to not use environment variables directly and pass the values in instead. I'll start with this new method.

I'll make these changes and share the commit with you to see what you think.

Reply:
https://github.com/hashicorp/boundary-ui/pull/2175/commits/75723dc98a84fb054d216fbe03c24772fb8762ca